ui/yarn.lock

```diff
@@ -7910,14 +7910,14 @@ react-dnd@^2.6.0:
     prop-types "^15.5.10"
 
 react-dom@^16.5.1:
-  version "16.5.2"
-  resolved "https://registry.yarnpkg.com/react-dom/-/react-dom-16.5.2.tgz#b69ee47aa20bab5327b2b9d7c1fe2a30f2cfa9d7"
-  integrity sha512-RC8LDw8feuZOHVgzEf7f+cxBr/DnKdqp56VU0lAs1f4UfKc4cU8wU4fTq/mgnvynLQo8OtlPC19NUFh/zjZPuA==
+  version "16.8.6"
```

Comment:
We should probably bump `react`, `react-dom`, `@types/react` and `@types/react-dom` all to "^16.8" in the package.json explicitly, rather than relying on Yarn's resolution.

Reply:
Will do